english/dream/code/bugs/bugs.md

```diff
@@ -0,0 +1 @@
+As I sat down to my laptop, I noticed that the keys had been transformed into little  beetles with white glyphs marked on their black carapaces. My fingers tapped down in their familiar rhythm, but the tiny black bugs scurried away before any input could be collected. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.